services/brig/src/Brig/API/User.hs

```diff
@@ -1615,3 +1623,24 @@ phonePrefixDelete = liftSem . BlacklistPhonePrefixStore.delete
 
 phonePrefixInsert :: Member BlacklistPhonePrefixStore r => ExcludedPrefix -> (AppT r) ()
 phonePrefixInsert = liftSem . BlacklistPhonePrefixStore.insert
+
+userUnderE2EId :: Member GalleyProvider r => UserId -> Sem r Bool
+userUnderE2EId uid = do
```

Comment:
```suggestion
userUnderE2EId uid =
```

Reply:
we've had this before.  :)  i like redundant do's because then i don't forget to add them during refactoring.  also is this bike shedding?

Reply:
Can't recall. No bike shedding. But an incredible nitpick. Totally fine to leave it. :)